mock-data/scraper.json

```diff
@@ -0,0 +1,13 @@
+{
+  "item": {
+    "id": "595750e2400327c53593f1b2",
+    "name": "medium-scraper",
+    "frequency": "0 0 0 * * *",
+    "apiUrl": "reader-medium-scraper.herokuapp.com",
```

Comment:
should have full url node domain. `https://reader-medium-scraper.herokuapp.com/v1/